api.swagger.yaml

```diff
@@ -835,6 +835,24 @@ paths:
               - asc
               - desc
             default: desc
+        - description: 'Filter by free skill'
```

Comment:
These 3 filters should be on UUID and allow for multiple.

- free_skill_uuid
- personal_interest_uuid
- organisational_skill_uuid

see example /content/list
        - name: owner_uuid
          description: 'Filter on Invotra UUID of the owner.'
          in: query
          schema:
            type: array
            items:
              type: string
              format: uuid
              example: '01234567-89ab-cdef-1234-56789abcdef0'

Also have sort & order at the end (we do this on all EPs)